client/components/NumberOfPeople.jsx

```diff
@@ -1,26 +1,24 @@
 import React from 'react';
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)